dotnet/Services/ProductReviewService.cs

```diff
@@ -742,6 +785,12 @@ public async Task<ReviewsResponseWrapper> GetReviews(int from, int to)
 
         public async Task<bool> DeleteReview(string[] ids)
         {
+            HttpStatusCode isValidAuthUser = await IsValidAuthUser();
+            if (isValidAuthUser != HttpStatusCode.OK)
+            {
+                return false;
```

Comment:
We should actually throw the error so that the user sees it (this applies to each of the mutations)